src/main/java/com/reshiftsecurity/plugins/intellij/common/util/SourceCodeUtil.java

```diff
@@ -72,7 +71,13 @@ public static boolean sourceFileExists(Path filePath) {
 
     public static Path getPathFromSourceLineAnnotation(SourceLineAnnotation annotation) {
         String packageName = annotation.getPackageName();
-        String packagePath = packageName.trim().replaceAll("[.]", File.separator);
+        String packagePath;
+        if(File.separator.equals("\\")){ //Edge case needed here due to how backslashes operate and the fact that Windows uses them
+            packagePath = packageName.trim().replaceAll("[.]", "\\\\");
+        }
+        else{
+            packagePath = packageName.trim().replaceAll("[.]", File.separator);
+        }
```

Comment:
Minor refactor, instead of duplicating the call `packagePath = packageName.trim().replaceAll("[.]", ...`
We could have a variable that holds the replace string defaulted to `File.separator`. And then replace it with one if statement. Example:
```
String separator = File.separator;
if(File.separator.equals("\\")){
    separator = "\\\\"
}
String packagePath = packageName.trim().replaceAll("[.]", separator);
```

Reply:
Hadn't thought of that, will implement